src/bin/driver.rs

```diff
@@ -456,19 +480,10 @@ mod app {
         // Todo: uncomment once we have Hardware
         // telemetry.monitor.header_temp =
         //     c.local.lm75.read_temperature().unwrap();
-        (telemetry.monitor.current, telemetry.monitor.voltage) =
-            c.shared.internal_adc.lock(|adc| {
-                (
-                    [
-                        adc.read_output_current(Channel::LowNoise),
-                        adc.read_output_current(Channel::HighPower),
-                    ],
-                    [
-                        adc.read_output_voltage(Channel::LowNoise),
-                        adc.read_output_voltage(Channel::HighPower),
-                    ],
-                )
-            });
+        telemetry.monitor.laser_interlock = c
+            .shared
+            .laser_interlock_pin
+            .lock(|pin| pin.get_state() == hal::gpio::PinState::High);
```

Comment:
You need to make sure it's clear what high and low means. Which is asserted? Either comment it everywhere or make an API that makes it clear.